includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -2466,7 +2466,7 @@ public function wc_setup_ready() {
 		$docs_url     = 'https://docs.woocommerce.com/documentation/plugins/woocommerce/getting-started/?utm_source=setupwizard&utm_medium=product&utm_content=docs&utm_campaign=woocommerceplugin';
 		$help_text    = sprintf(
 			/* translators: %1$s: link to videos, %2$s: link to docs */
-			__( 'Watch our <a href="%1$s" target="_blank">guided tour videos</a> to learn more about WooCommerce, and visit WooCommerce.com to learn more about <a href="%2$s" target="_blank">getting started</a>.', 'woocommerce' ),
+			__( 'Visit WooCommerce.com to learn more about <a href="%2$s" target="_blank">getting started</a>.', 'woocommerce' ),
 			$videos_url,
```

Comment:
I believe we can now remove `$videos_url` from the code (both here and a few lines above where it is defined) and then we will need to update the placeholder number in the string for `$docs_url` and the translators comment to reflect this change.

Reply:
Thanks for the input @rodrigoprimo - I've amended my [commit](https://github.com/woocommerce/woocommerce/pull/25823/commits/d2a00bff04835fd888ed0ef775e83a95fd54402e) to reflect this.